packages/edit-post/src/store/effects.js

```diff
@@ -40,8 +42,14 @@ const effects = {
 		//
 		// See: https://github.com/WordPress/WordPress/blob/5.1.1/wp-admin/includes/post.php#L2307-L2309
 		const hasActiveMetaBoxes = select( 'core/edit-post' ).hasMetaBoxes();
+
+		// First remove any existing subscription in order to prevent multiple saves
+		if ( isFunction( saveMetaboxUnsubscribe ) ) {
+			saveMetaboxUnsubscribe();
```

Comment:
Instead of not calling the function, wouldn't be better to "unsubcribe" the first function and resubscribe again as some global variables used inside the function could have changed and the latest version would be the most correct one?

Reply:
Sorry, I may not be quite understanding your comment, but I thought that this was the case, eg. if a subscription exists saveMetaboxUnsubscribe will be a function rather than undefined, in which case it will be run, essentially 'unsubscribing' before the resubscription is assigned to saveMetaboxUnsubscribe ?

Reply:
nevermind, I'm not awake properly :P You're doing exactly what I had in mind :)